content/300-accelerate/200-getting-started.mdx

```diff
@@ -20,7 +20,7 @@ To get started with Accelerate, you will need the following:
 
 ## 1. Enable Accelerate in a project
 
-In order to enable Accelerate, you can log in to [Prisma Data Platform](https://pris.ly/pdp) and create a new project. Follow the instructions in the UI to add Accelerate.
+In order to enable Accelerate, you can log in to [Prisma Data Platform](https://pris.ly/pdp) and create a new project. If your database uses IP allowlisting or you're setting up a firewall to restrict access to trusted IP addresses, ensure static IPs are enabled in your Accelerate configuration. To learn more about static IP see this [FAQ](/accelerate/faq#when-do-i-have-to-enable-static-ip-for-prisma-accelerate).
```

Comment:
I think this is not exactly accurate.

> ensure static IPs are enabled
This is not enough, it's rather a precondition, so that you can add the Accelerate IPs to your allowlist, right? Can we make this more clear?

Reply:
> This is not enough, it's rather a precondition, so that you can add the Accelerate IPs to your allowlist, right? Can we make this more clear?

How does the suggestion below look to you?

```suggestion
In order to enable Accelerate, you can log in to [Prisma Data Platform](https://pris.ly/pdp) and create a new project. If your database is using IP allowlisting to only allow traffic from trusted IP addresses, you'll need to provide a static IP address from which Accelerate will access your database. To obtain a static IP address for Accelerate, you'll need to enable static IP within your Accelerate configuration in your project environment. Then, you have to configure your database's firewall to permit traffic from the obtained static IP address, this will allow Accelerate to reach your database. To learn more about static IP see this [FAQ](/accelerate/faq#when-do-i-have-to-enable-static-ip-for-prisma-accelerate). To learn more about static IP see this [FAQ](/accelerate/faq#when-do-i-have-to-enable-static-ip-for-prisma-accelerate).
```

Reply:
> to only allow traffic from trusted IP addresses, you'll need to provide a static IP address from which Accelerate will access your database

This sounds confusing, I think we can vastly simplify. Suggestion:

```suggestion
In order to enable Accelerate, log in on the [Platform Console](https://console.prisma.io) and create a new project in your workspace. If your database has restrictions on the IP addresses that can connect to it, make sure static IP is enabled in your Accelerate configuration and use it to allow incoming traffic on your database. Static IP is available for workspaces on the Pro and Business plans. To learn more, take a look at the related [docs](/accelerate/faq#when-do-i-have-to-enable-static-ip-for-prisma-accelerate).
```

Reply:
@ankur-arch here is a [PR](https://github.com/prisma/docs/pull/5838) with updates to this section.